browser/resources/brave_new_tab_page_refresh/components/settings/settings_modal.tsx

```diff
@@ -33,14 +36,20 @@ interface Props {
 }
 
 export function SettingsModal(props: Props) {
+  const braveNews = useBraveNews()
   const searchFeatureEnabled = useSearchState((s) => s.searchFeatureEnabled)
 
   const [currentView, setCurrentView] =
       React.useState<SettingsView>(props.initialView || 'background')
 
   React.useEffect(() => {
     if (props.isOpen) {
-      setCurrentView(props.initialView || 'background')
+      if (props.initialView === 'news') {
+        braveNews.setCustomizePage('news')
+        setCurrentView('background')
+      } else {
+        setCurrentView(props.initialView ?? 'background')
+      }
```

Comment:
Just noticed that if I open the settings panel, click `Brave News` and press `Back to dashboard` the settings dialog is closed - I think clicking that back button should take me back to the settings as it does on the current NTP

Reply:
I'm not sure what is causing the difference, since the News modal doesn't actually re-show the settings dialog. There might be some difference in how the settings dialog is configured. Should we create a follow-up, or continue investigating here?

Reply:
Happy for it to be a follow up I think - it'd be nice to land this